robots.cc

```diff
@@ -66,7 +66,7 @@ class RobotsMatchStrategy {
   static bool Matches(absl::string_view path, absl::string_view pattern);
 };
 
-// Returns true iff URI path matches the specified pattern. Pattern is anchored
+// Returns true if URI path matches the specified pattern. Pattern is anchored
```

Comment:
This is probably intended to be "iff", for "[if and only if](https://en.wikipedia.org/wiki/If_and_only_if)".